src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -85,45 +85,45 @@
-    private static final int ROUTER_PORT = 3301;
-    private static final String CARTRIDGE_USERNAME = "admin";
-    private static final String CARTRIDGE_PASSWORD = "testapp-cluster-cookie";
-    private static final String DOCKERFILE = "Dockerfile";
-    private static final int API_PORT = 8081;
-    private static final String VSHARD_BOOTSTRAP_COMMAND = "return require('cartridge').admin_bootstrap_vshard()";
-    private static final String SCRIPT_RESOURCE_DIRECTORY = "";
-    private static final String INSTANCE_DIR = "/app";
-
-    private static final String ENV_TARANTOOL_VERSION = "TARANTOOL_VERSION";
-    private static final String ENV_TARANTOOL_SERVER_USER = "TARANTOOL_SERVER_USER";
-    private static final String ENV_TARANTOOL_SERVER_UID = "TARANTOOL_SERVER_UID";
-    private static final String ENV_TARANTOOL_SERVER_GROUP = "TARANTOOL_SERVER_GROUP";
-    private static final String ENV_TARANTOOL_SERVER_GID = "TARANTOOL_SERVER_GID";
-    private static final String ENV_TARANTOOL_WORKDIR = "TARANTOOL_WORKDIR";
-    private static final String ENV_TARANTOOL_RUNDIR = "TARANTOOL_RUNDIR";
-    private static final String ENV_TARANTOOL_LOGDIR = "TARANTOOL_LOGDIR";
-    private static final String ENV_TARANTOOL_DATADIR = "TARANTOOL_DATADIR";
-    private static final String ENV_TARANTOOL_INSTANCES_FILE = "TARANTOOL_INSTANCES_FILE";
-    private static final String ENV_TARANTOOL_CLUSTER_COOKIE = "TARANTOOL_CLUSTER_COOKIE";
-    private static final String healthyCmd = "return require('cartridge').is_healthy()";
-    private  static final int TWO_MINUTES = 120;
-
-    private final CartridgeConfigParser instanceFileParser;
-    private final TarantoolContainerClientHelper clientHelper;
-    private final String TARANTOOL_RUN_DIR;
-
-    private boolean useFixedPorts = false;
-    private String routerHost = ROUTER_HOST;
-    private int routerPort = ROUTER_PORT;
-    private int apiPort = API_PORT;
-    private String routerUsername = CARTRIDGE_USERNAME;
-    private String routerPassword = CARTRIDGE_PASSWORD;
-    private String directoryResourcePath = SCRIPT_RESOURCE_DIRECTORY;
-    private String instanceDir = INSTANCE_DIR;
-    private String topologyConfigurationFile;
-    private String instancesFile;
-    private SslContext sslContext;
+    protected static final String ROUTER_HOST = "localhost";
+    protected static final int ROUTER_PORT = 3301;
+    protected static final String CARTRIDGE_USERNAME = "admin";
+    protected static final String CARTRIDGE_PASSWORD = "testapp-cluster-cookie";
+    protected static final String DOCKERFILE = "Dockerfile";
+    protected static final int API_PORT = 8081;
+    protected static final String VSHARD_BOOTSTRAP_COMMAND = "return require('cartridge').admin_bootstrap_vshard()";
+    protected static final String SCRIPT_RESOURCE_DIRECTORY = "";
+    protected static final String INSTANCE_DIR = "/app";
+
+    protected static final String ENV_TARANTOOL_VERSION = "TARANTOOL_VERSION";
```

Comment:
Can we make public constants with the ENV_ prefix for ease of use when you add them as Dockerfile build args?

Reply:
Good idea

Revealed them